ui/src/app/components/workspace/workspace.component.scss

```diff
@@ -9,7 +9,7 @@
     button {
       border: none;
       background-color: inherit;
-      color: #1967d2;
+      color: #3367D6;
```

Comment:
Remove color assignment from here. Instead try to reuse **.mat-button.mat-primary** define in global css

Reply:
Made the suggested changes

Reply:
Made the suggested changes